### PR TITLE
Request to add es-theme-ComiBook and es-themeComicBook_4-3

### PR DESCRIPTION
**Pull request by TMNTturtleguy**
I appologize for my early request to the main branch.  Herb suggested I start using the desktop gui for git and i made an error on proper modifications.  I do appologize.

### CODE REVIEW

**Comment by HerbFargus** [comment]
thanks, you'll need to fix the formatting so that the spaces are the same as the others before this can be merged (you can use an editor like notepad++ to view whitespace)

**Comment by TMNTturtleguy** [comment]
I appreciate your time and mentor ship today.  I apologize however i am not sure what spaces in what file i need to fix?  I do use notepad ++ for all of my editing.  In the  esthemes.sh it looks like my formatting is the same as the others.  Thank you again.

**Comment by HerbFargus** [comment]
You can see the difference here:

https://github.com/TMNTturtleguy/RetroPie-Setup/blob/bf9ab49f0af5a7a499cddce073ed6581691d32ee/scriptmodules/supplementary/esthemes.sh#L88-L89

It looks like you used two tabs instead of 8 spaces.

**Comment by TMNTturtleguy** [comment]
interesting, you are correct, for some reason on my screen it lined up correctly, it is fixed.  Not I did it from the desktop and now can't figure out why it made a separate commit instead of updating the first, and I can't delete the first commit. I have a lot to learn.  Here is what I see on my computer with 2 tabs.  Interesting.
image

File: scriptmodules/supplementary/esthemes.sh [hunk 1]
```diff
@@ -85,6 +85,8 @@ function gui_esthemes() {
         'ruckage famicom-mini'
         'anthonycaccese crt'
         'anthonycaccese crt-centered'
+        'TMNTturtlguy ComicBook'
+        'TMNTturtlguy ComicBook_4-3'
```
Comment (HerbFargus) [inline scriptmodules/supplementary/esthemes.sh]: looks like your github username is TMNTturtleguy (missing an e)

**Comment by HerbFargus** [comment]
I made a note on another typo that needs to be fixed as well.

as far as commits go, yes, it will create a new commit for any edits you push after your last commit. There is a thing called  squashing that allows you to squash all your commits into the first one, but I only know how to do it through the command line (I haven't looked at the desktop gui if thats  a feature of it or not) I left notes on how for meleu on his other PR https://github.com/RetroPie/RetroPie-Setup/pull/1980#issuecomment-302953135

**Comment by TMNTturtleguy** [comment]
Thanks, i will read up on how to squash.  Where did you make a note on another typo?  I don't see your note, and I went through the styling guide as well.  The only thing i can see is in my name, which is a long story, but when i opened up my forum account, it was on my phone and i missed the 'e' in turtle.  When i never noticed it until after i opened my account on here, now they are different.  Is that the typo?  Thanks

**Comment by HerbFargus** [comment]
@TMNTturtleguy oops, looks like I still have a few things to learn about github too (this code review stuff is a newer thing) typically I just complain in the comments but I guess it makes sense to use the tools github has available. now it should show up but yes it was the E thing

File: scriptmodules/supplementary/esthemes.sh (hunk 1 above)
Comment (TMNTturtleguy) [inline scriptmodules/supplementary/esthemes.sh]: I will change to match my github name.  Thanks

**Comment by HerbFargus** [comment]
If you want to figure out how to squash the commits I'll defer to jools for merging once that's complete. looks good though, just tested the theme out too for the first time. very impressive!

**Comment by HerbFargus** [comment]
@TMNTturtleguy I've squashed and merged it on our side. so its included now. thanks for your work on this!

**Comment by TMNTturtleguy** [comment]
Thank you so much! I have been spending the last hour trying to figure out the git gui and git bash just to get to the starting point of squashing!  I really appreciate your help, and the kind words regarding the theme.  It was a lot of fun to make.

**Pull request merged**